x/concentrated-liquidity/math/math.go

```diff
@@ -58,19 +58,30 @@ func CalcAmount0Delta(liq, sqrtPriceA, sqrtPriceB sdk.Dec, roundUp bool) sdk.Dec
 		sqrtPriceA, sqrtPriceB = sqrtPriceB, sqrtPriceA
 	}
 	diff := sqrtPriceB.Sub(sqrtPriceA)
-	denom := sqrtPriceA.Mul(sqrtPriceB)
 	// if calculating for amountIn, we round up
-	// if calculating for amountOut, we don't round at all
+	// if calculating for amountOut, we don't round but truncate at precision end
+	// in the rounding direction.
```

Comment:
```suggestion
	// if calculating for amountOut, we round down
```

Reply:
These two statements are equivalent right?

Reply:
What I wanted to say is that we round down at the precision end, not to the nearest integer. Let me modify your suggestion and commit the update